test/conformance/api/v1/service_test.go

```diff
@@ -318,8 +318,17 @@ func TestServiceWithTrafficSplit(t *testing.T) {
 	if err := validateDomains(t, clients,
 		names.URL,
 		[]string{expectedFirstRev},
-		[]string{"latest", "current"},
-		[]string{expectedFirstRev, expectedFirstRev}); err != nil {
+		[]trafficExpectedPair{
+			{
+				traffic:  "latest",
+				expected: expectedFirstRev,
+			},
+			{
+				traffic:  "current",
+				expected: expectedFirstRev,
+			},
+		},
```

Comment:
```suggestion
		[]trafficExpectedPair{{
			traffic:  "latest",
			expected: expectedFirstRev,
		},{
			traffic:  "current",
			expected: expectedFirstRev,
		}},
```

nit (applicable throughout), to spare some indentation depth.